src/algorithms/tracking/TrackProjector.cc

```diff
@@ -41,13 +42,13 @@ namespace eicrecon {
 
 
     void TrackProjector::process(const Input& input, const Output& output) const {
-        const auto [trajectories] = input;
+        const auto [acts_trajectories, tracks] = input;
         auto [track_segments] = output;
 
-        debug("Track projector event process. Num of input trajectories: {}", std::size(trajectories));
+        debug("Track projector event process. Num of input trajectories: {}", std::size(acts_trajectories));
 
         // Loop over the trajectories
-        for (const auto &traj: trajectories) {
+        for (size_t i = 0; const auto &traj: acts_trajectories) {
```

Comment:
Avoid `stddef.h` C-header by using C++ type?
```suggestion
        for (std::size_t i = 0; const auto &traj: acts_trajectories) {
```